ntbs-service/Pages/Notifications/Edit/_SaveButton.cshtml

```diff
@@ -1,21 +1,39 @@
+@using ntbs_service.Helpers
 @using ntbs_service.Models.Enums
 @model ntbs_service.Models.Entities.Notification
 
 @{
+    var isDraft = Model.NotificationStatus == NotificationStatus.Draft;
+    var isNonEditPage = (bool?) ViewData["IsNonEditPage"] ?? false;
```

Comment:
I think it'll read easier as `isEditPage`.. unless you want to blame it on me not having gone through my coffee this morning yet, in which case, fine ;P